v3/UNRELEASED_CHANGELOG.md

```
@@ -23,6 +23,7 @@ After processing, the content will be moved to the main changelog and this file
 
 ## Fixed
 <!-- Bug fixes -->
+- Fixed Wayland window size maximising issues (https://github.com/wailsapp/wails/issues/4429) by [@samstanier](https://github.com/samstanier)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)